easybuild/easyconfigs/o/OpenPhase/OpenPhase-0.9.1-foss-2017b.eb

```diff
@@ -0,0 +1,51 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'OpenPhase'
+version = '0.9.1'
+
+homepage = 'http://www.openphase.de/'
+
+description = """
+ OpenPhase is the open source software project targeted at the phase field
+ simulations of complex scientific problems involving microstructure formation
+ in systems undergoing first order phase transformation.
+"""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+# http://www.openphase.de/upload/content/OpenPhase.V0.9.1.zip
+source_urls = ['http://www.openphase.de/upload/content/']
+sources = ['%(name)s.V%(version)s.zip']
+checksums = ['9bbfb2ddcb405246325f9171e8fe21375488ba560c40b4a338395e406d88c3f6']
+
+builddependencies = [
+    ('Doxygen', '1.8.14'),
+]
+
+skipsteps = ['configure']
+
+start_dir = '..'
+buildininstalldir = True
+
+prebuildopts = """
+    mv -v %(name)s.V%(version)s/* .
+    rmdir -v %(name)s.V%(version)s
+    echo -e "\ninstall:\n" >> Makefile  # dummy install so we can use postinstallcmds
```

Comment:
@JackPerdue This looks a bit... fishy. Can you clarify what's going on here?

Reply:
@JackPerdue ping on the query from @boegel ?